application/utils.py

```diff
@@ -0,0 +1,44 @@
+from constans.constans import EAST, NORTH, SOUTH, WEST
+from constans.constants_game import LARGE
+from game.board import Board
+from game.character import Character
+from game.player import Player
+
+
+def is_a_player_character(row, col, current_player: Player, board: Board) -> bool:
+    cell = board.get_cell(row, col)
+    character: Character = cell.character
+    return character.player.user_name == current_player.user_name if character else False
```

Comment:
NIT: As far as i understand, character.player.user_name should be a String. 
In the second part of the expression, if there's a character we compare it with the current player's user_name (which seems to be correct), but if there's no character we compare it with a False which doesn't seems to be the fanciest solution 😬 
Regardless IMHO it will work, consider improving this expression.

Reply:
Yes, it doesn't seems right. I fixed in [66a45ef](https://github.com/evbeda/edagames-wumpus/commit/66a45eff499eafebbc87846fc39f99ba60d42d59) commit. Thanks for the feedback!